### ISSUE TITLE
Min deployment target 10+

### ISSUE THREAD

**Issue by carmelogallo** [makomori/Dots#2]
Hi guys, one question for you. Min deployment target 10+, just why?

**Comment by makomori** [comment]
@cikpis This library is using UIViewPropertyAnimator for animations and its requirement is iOS 10. That's why. We can use this lib in iOS 9 or 8, if we also write the animation by using old UIView animation. But it's not written yet, sorry.

**Comment by carmelogallo** [comment]
Yes I know why, I read the code :) My question was why didn't you implement iOS 9 side as well. It's still an important version.

Thanks anyway, cheers.
